docs/calico-with-docker/without-docker-networking/ManualSetup.md

```diff
@@ -18,8 +18,14 @@ the example.
 
 ## Requirements
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+You will need 2 servers (bare metal or VMs) with a modern 64-bit Linux OS and
+IP connectivity between them.  The servers must not have any software running
+that tries to manage or interfere with new interfaces that appear (for example
+adding or removing routes or iptables through them, or modifying interface
```

Comment:
I'm worried about "through them" - it didn't mean to me what it means to you, and I'm a native english speaker.

Perhaps: "The servers must not run any software that tries to manage or interfere with new interfaces or related config (for example routes, iptables, ipsets or interface state)"